src/Native/Unix/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -12,8 +12,19 @@ add_library(System.IO.Compression.Native
     ${VERSION_FILE_PATH}
 )
 
+add_library(System.IO.Compression.Native-Static
+    STATIC
+    ${NATIVECOMPRESSION_SOURCES}
+    ${VERSION_FILE_PATH}
+)
+
+# Disable the "lib" prefix and override default name
+set_target_properties(System.IO.Compression.Native-Static PROPERTIES PREFIX "")
```

Comment:
Could you please update the existing src\Native\Unix\System.Native\CMakeLists.txt to use same formatting?